vnext/Microsoft.ReactNative/Fabric/Composition/Modal/WindowsModalHostViewComponentView.cpp

```diff
@@ -95,7 +100,7 @@ void WindowsModalHostComponentView::EnsureModalCreated() {
   m_hwnd = CreateWindow(
       c_modalWindowClassName,
       L"React-Native Modal",
-      WS_OVERLAPPEDWINDOW,
+      (WS_POPUP),
```

Comment:
Is it this that causes the titlebar to disappear?

Reply:
yes! And another line for adjusting the size for the titlebar but just added them behind m_showTitleBar!